lib/github.com/mpllang/mpllib/AugMap.sml

```diff
@@ -75,9 +75,7 @@ struct
 
     fun singleton_weight () = 1
 
-    val ratio =
-      case T.balance of
-        WB (x) => x / (1.0 - x)
+    val ratio = let val WB x = T.balance in x / (1.0 - x) end
```

Comment:
I prefer the single-branch `case` here because the idea was that `T.balance` could be extended with other balancing criteria in the future.